test/rpc-test.js

```diff
@@ -0,0 +1,227 @@
+/* eslint-env mocha */
+/* eslint prefer-arrow-callback: "off" */
+
+'use strict';
+
+const assert = require('./util/assert');
+const consensus = require('../lib/protocol/consensus');
+const FullNode = require('../lib/node/fullnode');
+const Network = require('../lib/protocol/network');
+const network = Network.get('regtest');
+
+const node = new FullNode({
```

Comment:
This should use a unique port that won't conflict with an already running regtest node.

Reply:
Tests will not pass in this case:
```
./bin/bcoin --network=regtest
npm run test-file test/rpc-test.js
```

Reply:
This is an existing issue with other tests https://github.com/bcoin-org/bcoin/issues/717 though we can fix this one now.

Reply:
Fixed the others here https://github.com/bcoin-org/bcoin/pull/719